### PR TITLE
Feature/kx1 fe development

### PR DESCRIPTION
**Pull request by SaraPoiatti**

### CODE REVIEW

File: code/client/src/API/API.ts [hunk 1]
```diff
@@ -45,10 +48,58 @@ async function getUserInfo() {
...
+        }
+      })
+      .catch((err: any) => reject({ error: "Cannot communicate" })); // connection error
+  });
+}
+/**
+ * This funciton adds a new page in db.
+ */
+function addDocument(document: NewDocument) {
+  console.log(document);
```
Comment (wolf1128) [inline code/client/src/API/API.ts:84]: Please remove the console.log

File: code/client/src/modules/UrbanPlanner/AddDocumentForm/AddDocumentForm.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            language: "",
+            pages: "",
+            georeference: [[]],
+          };
+          props.setDocument(newDoc);
+          navigate("/urban-planner");
+          setErrorMessage("");
+        })
+        .catch((error) => {
+          console.error("Error adding document:", error);
```
Comment (wolf1128) [inline code/client/src/modules/UrbanPlanner/AddDocumentForm/AddDocumentForm.tsx:63]: please comment this for debugging purposes.

File: code/client/src/components/LeftSideBar.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  return (
+    <div className="sidebar-container">
+      <Row className="top-side-box">
+        {user ? (
+          <Col>
+            <div className="menu-text">
+              <span className="file-icon-box">
+                <i className="bi bi-files"></i>
+              </span>
```
Comment (wolf1128) [inline code/client/src/components/LeftSideBar.tsx:29]: Let put ourself in the shoes of the urban planner!
When you clicked on the "new document" mistakenly and want to go back to the main page of the dashboard, what do you do?
I think There should be a "Dashboard" link or sth above the "new document" maybe.

Comment (SaraPoiatti) [reply code/client/src/components/LeftSideBar.tsx:29]: Right now he can use "Cancel" button to go back to dashboard.

**Pull request merged**